todolist/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.urls import reverse
+
+
+class toDoList(models.Model):
```

Comment:
In python we usually use ThisCamelCase for classes

Reply:
go u